task/06-conditions-n-loops-tasks.js

```diff
@@ -192,10 +217,27 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+  let interval = "";
+  if (b < a) {
+    let c = a;
+    a = b;
+    b = c;
+  }
+  if (isStartIncluded) {
+    if (isEndIncluded) {
+      interval = "[" + a + ", " + b + "]";
+    } else {
+      interval = "[" + a + ", " + b + ")";
+    }
+  } else if (isEndIncluded) {
+    interval = "(" + a + ", " + b + "]";
+  } else {
+    interval = "(" + a + ", " + b + ")";
+  }
+
+  return interval;
 }
```

Comment:
Can be more simply, try to simplify the solution to an one line using `template literals` and `ternary operator`